tests/files_test_endl/testfile

```diff
@@ -0,0 +1,2545 @@
+ 82000  9  0  0.00000+ 0 8912015 2  0.00000+ 0  0.00000+ 0  0.00000+ 0  
+ 7  0  0  0.00000+ 0  0.00000+ 0  0.00000+ 0  0.00000+ 0  0.00000+ 0
```

Comment:
I started from an EPDL data file (the one downloaded by the unit test), removed some tables and randomized the data in the remaining ones. I guess it's OK?